@here/olp-sdk-dataservice-read/lib/DataStoreDownloadManager.ts

```diff
@@ -49,25 +49,23 @@ class DeferredPromise<T> {
 }
 
 /**
- * `DataStoreDownloadManager` for downloading URLs.
+ * Downloads URLs.
```

Comment:
not URLs, This is an implementation of DownloadManager interface requested by datastore-api module for fetching data from the backend or sending requests to the backend. This implementation allows us  to fetch blobs or JSONs data with aborting or re-sending (on timeouts) mechanisms with limiting for maximum parallel requests

Reply:
Fixed

Reply:
Fixed